script/zip-comp-logs

```diff
@@ -4,27 +4,53 @@ A script to zip the logs produced by a series of competition matches.
 """
 
 import re
-import shutil
 import argparse
 import datetime
 from typing import List
 from pathlib import Path
 from zipfile import ZipFile, ZIP_DEFLATED
 
 
-def zip_folder(folder: Path, output_dir: Path, suffix: str, include_date: bool) -> str:
+def match_animation_files(log_name: str, archives_dir: Path) -> List[Path]:
+    match_num_search = re.search(r'match-([0-9]+)', log_name)
+    if not isinstance(match_num_search, re.Match):
+        print(f'Invalid match name: {log_name}')
```

Comment:
Suspect this is going to need a `noqa:T001` unless we've specifically turned it off for this file.